qiskit_ibm_runtime/api/clients/runtime.py

```diff
@@ -131,6 +132,7 @@ def program_run(
             image: The runtime image to use.
             hgp: Hub/group/project to use.
             log_level: Log level to use.
+            session_id: Job ID of the first job in a runtime session.
```

Comment:
shouldn't the session ID exist independent of any jobs in that session?

Reply:
eventually I think we will get there when the API supports it, but for now the session starts only when the first job is run and that first job id is used as the session id...